src/components/about-us/utils/lottie.js

```diff
@@ -29,15 +29,17 @@ export default class Lottie extends PureComponent {
     this.registerEvents(eventListeners)
   }
 
-  componentWillUpdate(nextProps /* , nextState */) {
+  shouldComponentUpdate(nextProps /* , nextState */) {
     /* Recreate the animation handle if the data is changed */
     if (this.options.animationData !== nextProps.options.animationData) {
       this.deRegisterEvents(this.props.eventListeners)
       this.destroy()
       this.options = { ...this.options, ...nextProps.options }
       this.anim = lottie.loadAnimation(this.options)
       this.registerEvents(nextProps.eventListeners)
+      return true
     }
+    return false
```

Comment:
I think `shouldComponentUpdate` is used to check the differences between `this.(props|state)` and `(next(Props|State))`.
Should we avoid doing something like `Ln.35 -> Ln.39`?

Reply:
Updated